### PR TITLE
Angular v.8 rc4 packages

### PR DESCRIPTION
**Pull request by bogacg**
Please review as a preparation to Angular v.8 launch.
I did update package versions, test, build and had some warnings/errors therefore made some minor changes accordingly.

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -61,9 +61,9 @@ if (shell.exec(`rollup -c rollup.es.config.js -i ${NPM_DIR}/${PACKAGE}.js -o ${F
 }
 
 shell.echo(`Produce ESM5/FESM5 versions`);
-shell.exec(`ngc -p ${OUT_DIR}/tsconfig-build.json --target es5 -d false --outDir ${OUT_DIR_ESM5} --sourceMap`);
+shell.exec(`tsc -p ${OUT_DIR}/tsconfig-build.json --target es5 -d false --outDir ${OUT_DIR_ESM5} --sourceMap`);
```
Comment (robisim74) [inline build.js:64]: Move to typescript compiler is not correct. The error is in tsickle ^0.33.0 https://github.com/angular/tsickle/issues/930

**Comment by robisim74** [comment]
@bogacg Thanks for this pr. But as I said, I'll release a new version when official Angular v8 will be released. However, I will consider the modifications of this pr.

File: build.js (hunk 1 above)
Comment (bogacg) [reply build.js:64]: I didn't know what is the exact cause of that error, you know better 👍

**Comment by bogacg** [comment]
Yes, waiting for official release is the right move.

**Comment by robisim74** [comment]
I'm closing this pr, since I updated directly keeping it in mind. Thanks again!

**Pull request closed without merging**